### DISCUSSION TITLE
Show numbers of subcomments in not fully collapsed comments without toolbar

### DISCUSSION THREAD

**Discussion by Liquid-Aristocracy** [Docile-Alligator/Infinity-For-Reddit#1144] [Ideas]
I like to browse comments with no toolbar for a more compact layout. However without toolbar, it's unable to tell whether a comment has subcomments after I collapse it.

**Comment by pfr-dev** [comment]
100% need this feature

**Comment by Dragonfly-30** [comment]
Yeah it will be good to have it. I also get the same problem.
